File: OOPHP/Persoon.py
class Persoon:
  def __init__(self, naam, leeftijd, geslacht, is_student, gemiddeld_cijfer):
    self.naam = naam
    self.leeftijd = leeftijd
    self.geslacht = geslacht
    self.is_student = is_student
    self.gemiddeld_cijfer = gemiddeld_cijfer

  def __str__(self):
    return f"Naam: {self.naam}\nLeeftijd: {self.leeftijd}\nGeslacht: {self.geslacht}\nStudent: {'Ja' if self.is_student else 'Nee'}\nGemiddeld cijfer: {self.gemiddeld_cijfer}"

persoon1 = Persoon("Stefan Milovic", 25, "M", True, 7.2)
persoon2 = Persoon("Joey Fleming", 30, "M", False, 6.5)
persoon3 = Persoon("Poka Hontas", 22, "V", True, 8.1)

print(persoon1)
print(persoon2)
print(persoon3)

File: OOPHP/Persoon.php
<?php 
class Persoon {
    private $naam;
    private $leeftijd;
    private $geslacht;
    private $isStudent;
    private $gemiddeldCijfer;
  
    public function __construct($naam, $leeftijd, $geslacht, $isStudent, $gemiddeldCijfer) {
      $this->naam = $naam;
      $this->leeftijd = $leeftijd;
      $this->geslacht = $geslacht;
      $this->isStudent = $isStudent;
      $this->gemiddeldCijfer = $gemiddeldCijfer;
    }
}

 $persoon1 = new Persoon("Stefan Milovic", 25, "M", true, 7.2);
 $persoon2 = new Persoon("Joey Fleming", 30, "M", false, 6.5);
 $persoon3 = new Persoon("Poka hontas", 22, "V", true, 8.1);

 echo $persoon1 . "\n";
 echo $persoon2 . "\n";
 echo $persoon3 . "\n";